src/seedu/addressbook/Main.java

```diff
@@ -104,13 +111,16 @@ private void recordResult(CommandResult result) {
      *
      * @param command user command
      * @return result of the command
+     * @throws StorageOperationException if there are errors writing to the storage file
      */
-    private CommandResult executeCommand(Command command)  {
+    private CommandResult executeCommand(Command command) throws StorageOperationException {
```

Comment:
As discussed during tutorial, the explicit `throws StorageOperationException` should be on the lower levels. 

If its caught and handled in this method, it does not `throws StorageOperationException`